pkg/odo/cli/component/delete.go

```diff
@@ -267,6 +267,7 @@ func NewCmdDelete(name, fullName string) *cobra.Command {
 
 	componentDeleteCmd.Flags().BoolVar(&do.showLogFlag, "show-log", false, "If enabled, logs will be shown when deleted")
 
+	componentDeleteCmd.Annotations["command"] = "main"
```

Comment:
Can you also add this to `odo delete`(v3) command?

Reply:
sure, looks like it was just recently merged in. I'll rebase